vj4/ui/pages/discussion_create.page.js

```diff
@@ -0,0 +1,86 @@
+import { NamedPage } from 'misc/PageLoader';
+import 'simple-cmeditor/dist/simplemde.min.css';
+
+const page = new NamedPage('discussion_create', () => {
+
+  require.ensure([
+    'simple-cmeditor/dist/simplemde.min.js',
+  ], (require) => {
+    const SimpleMDE = require('simple-cmeditor/dist/simplemde.min.js');
+
+    new SimpleMDE({
```

Comment:
VJMDE should also support at least `focus`, `setText`, `insertText`, `appendTo`, in order to be used by [CommentBox] component (https://github.com/vijos/vj4/blob/master/vj4/ui/components/discussion/CommentBox.js)